cypress/integration/publisherCanPublishArticles.feature.js

```diff
@@ -0,0 +1,81 @@
+describe("Editor can", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles",
+      response: "fixture:side_articles_shown.json"
+    });
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles/**",
+      response: "fixture:article_show.json"
+    });
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/v1/auth/sign_in",
+      response: "fixture:login_publisher.json"
+    });
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/auth/**",
+      response: "fixture:login_publisher.json"
+    })
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/admin/articles",
+      response: "fixture:publisher_articles.json"
+    });
+    cy.route({
+      method: "put",
+      url: "http://localhost:3000/api/v1/admin/articles/1?published=true",
+      response: "fixture:published_response.json"
+    });
+    cy.visit("/admin");
+    cy.get("#loginButton").click();
```

Comment:
Only use `kebab-case` for html tag id's, please

Reply:
fixed